Tabloid/client/src/modules/postManager.js

```diff
@@ -0,0 +1,5 @@
+const baseUrl = "/api/post";
+
+export const GetAllPublishedPosts = () => {
+  return fetch(baseUrl).then((res) => res.json());
```

Comment:
you're going to have to use the `getToken()` function here if you've added the `Authorize` attribute to the PostController in your API.

Reply:
Thank you for pointing that out. All is resolved and the code is working correctly now.